media-sound/audiotag/audiotag-0.19-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Command-line tool for mass tagging/renaming of audio files"
+HOMEPAGE="https://github.com/Daenyth/audiotag"
+SRC_URI="https://github.com/downloads/Daenyth/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="aac flac mp3 vorbis"
```

Comment:
Can these be moved to `optfeature` since they don't do anything in this ebuild?
(@gyakovlev)

Reply:
sounds reasonable, i've updated the ebuild to use ```optfeature``` instead.